Scripts/script-AssignAnalystToIncident.yml

```diff
@@ -28,37 +34,60 @@ script: |
 
   assignBy = args.assignBy || 'random';
 
+  function chooseUser(usersRes){
```

Comment:
add space before the `{`

Reply:
change function name to `pickRandomUser`

Reply:
Done